examples/src/main/java/com/mapbox/navigation/examples/core/InstructionViewActivity.kt

```diff
@@ -285,7 +286,9 @@ class InstructionViewActivity : AppCompatActivity(), OnMapReadyCallback,
         screenshotView.visibility = VISIBLE
         screenshotView.setImageBitmap(snapshot)
         mapView.visibility = View.INVISIBLE
-        val encodedSnapshot = ViewUtils.encodeView(ViewUtils.captureView(mapView))
+        val encodedSnapshot = ViewUtils.encodeView(ViewUtils.captureView(mapView),
+                BitmapEncodeOptions.Builder()
+                        .width(400).compressQuality(40).build())
```

Comment:
```suggestion
        val encodedSnapshot = ViewUtils.encodeView(ViewUtils.captureView(mapView),
                BitmapEncodeOptions(width = 400, compressQuality = 40))
```

Reply:
Or use the Builder. It helps if you ever add parameters to BitmapEncodeOptions. 

But is really helpful for external customers. Makes it so we won't introduce as many SEMVER

Reply:
@abhishek1508 had concern of this that the `data class` with public `constructor` negates the Builder pattern.
https://github.com/mapbox/mapbox-navigation-android/pull/3202#discussion_r443844768

Reply:
Yeah I trust you all, end of the day it will work so it's not a blocking issue. I'm personally going to use the data class constructor when I want my code to break when new values are added, because that's what will happen.

When using the builder pattern, new values and new defaults will be automatically applied. 

We want external users of our SDK to always use the Builder pattern. Understanding the concern for consistency, so am open to making a rule. But if you want the Builder to be required for your data object, add the Builder to the constructor 😄

Reply:
👍  to Builder here.